### ISSUE TITLE
How to make sigma profile graph through openCOSMO-RS_py

### ISSUE THREAD

**Issue by sbkk000** [TUHH-TVT/openCOSMO-RS_py#6]
The sigma profile.exe file is no longer available for download from the original webpage. 
How can we generate a sigma profile graph using openCOSMO-RS_py?
It will average the surface charge densities from the *.cosmo file to find an effective surface charge density on a standard surface segment.

![image](https://github.com/TUHH-TVT/openCOSMO-RS_py/assets/35765618/38889343-cefd-45f5-b03e-16235c4bb15a)

$\sigma_m=\frac{\sum_n\sigma_n^*\frac{r_n^2r_{a\nu}^2}{r_n^2+r_{a\nu}^2}\exp\left(-\frac{d_{mn}^2}{r_n^2+r_{a\nu}^2}\right)}{\sum_n\frac{r_n^2r_{a\nu}^2}{r_n^2+r_{a\nu}^2}\exp\left(-\frac{d_{mn}^2}{r_n^2+r_{a\nu}^2}\right)}$

**Comment by simonmb** [comment]
Hi, we have never provided a sigma profile executable to draw sigma profiles. You can however load a sigma profile on the python version of openCOSMO-RS or use the input parser from the conformer pipeline to load it. After loading it you could plot it using e.g.matplotlib.

**Comment by sbkk000** [comment]
Yes, sigmaprofil.exe was downloaded from this website https://apps.che.vt.edu/Liu-2013/VT-Databases.html Unfortunately, it is no longer possible to download now.
May I ask how to use openCOSMO-RS to process the data in the orcacosmo file and obtain the data in the sigma profile? 
sigma profile X-axis is $\sigma_m $, and its Y-axis is $p (\sigma) A_i $.
There is no relevant demonstration for examples.

**Comment by simonmb** [comment]
I am sorry, this link is probably from Virginia Tech University. This repository is from Hamburg University of Technology in Germany. We don't provide an executable for plotting sigma profiles. But you can do it as described above.
